scripts/library_handler.py

```diff
@@ -171,6 +171,16 @@ def findLibAndVersion(lib, version=None):
     elif lib == 'AMSC':
       # FIXME improve AMSC setup.py so it's compatible with importlib_metadata!
       return findLibAndVersionSubprocess('AMSC')
+    elif lib == 'PIL':
+      try:
+        import PIL
+        found = True
+        foundVersion = PIL.__version__
+        output = 'Library found.'
+      except impErr:
+        found = False
+        foundVersion = None
+        output = 'Library not found.'
```

Comment:
you can use 
```
return findLibAndVersionSubprocess("PIL")
```

Reply:
see method below

Reply:
Hm, findLibAndVersionSubprocess is a bit slower (since it uses a subprocess), but I doubt it matters for PIL.

Reply:
oh okay...should we change it directly? and make ti faster?

Reply:
Sure, I can make a non-subprocess version.

Reply:
For what it is worth, #1210 already changed this to findLibAndVersionSubprocess.